charts/geoserver/v0.3.2/values.yaml

```diff
@@ -0,0 +1,132 @@
+image:
+  registry: docker.io
+  repository: kartoza/geoserver
+  tag: "2.17.0"
```

Comment:
I think we need to upgrade this to the latest version, Currently, GeoServer is at 2.21.0 but I haven't pushed the latest version to hub so maybe 2.20.4

Reply:
Tag updated.